CMakeLists.txt

```diff
@@ -34,8 +34,10 @@ function (superbuild_find_projects var)
     embree
     exodus
     f3d
+    hdf5
```

Comment:
why are these new deps required?

Reply:
exodus dummy project uses hdf5 and netcdf, it used to use the one available in the sources. 

That was an oversight and it was fixed and now rely on the one built by the superbuild.

Reply:
szip is a depenency of hdf5.